docs/api/app.md

```diff
@@ -760,6 +760,23 @@ Disables hardware acceleration for current app.
 
 This method can only be called before app is ready.
 
+### `app.getAppMemoryInfo()`
+
+Returns `Object[]`:
```

Comment:
This should be a new structure, maybe `ProcessMemoryInfo[]`

Then the object described below should be moved to a new structure markdown file 👍